MapboxNavigation.podspec

```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "MapboxMaps", "10.8.0-rc.1"
+  s.dependency "MapboxMaps", "~> 10.8.0"
```

Comment:
Let's use `~> 10.8` instead of `~> 10.8.0`, it'll allow us to resolve Maps SDK to `10.9` and higher if it goes out.

Reply:
Thanks for the suggestion, this issue is fixed.